### PR TITLE
[16.0] [MIG] delivery_dropoff_site

### PR DESCRIPTION
**Pull request by jdoutreloux**
Based on https://github.com/OCA/delivery-carrier/pull/504

### CODE REVIEW

**Review by ThomasBinsfeld — APPROVED** [review]

**Comment by DorianMAG** [comment]
Thx for this work,
lgtm !

**Review by DorianMAG — APPROVED** [review]
lgtm

**Comment by jdoutreloux** [comment]
> Thx for this work, lgtm !

thanks :)

**Review by gaelTorrecillas — APPROVED** [review]

**Comment by gaelTorrecillas** [comment]
hello, 

Thanks you for your job.

**Review by flotho — APPROVED** [review]
functionnal test OK on ruboat

File: delivery_dropoff_site/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                carrier.with_dropoff_site != partner.is_dropoff_site
+                or (
+                    carrier.with_dropoff_site
+                    and partner.dropoff_site_carrier_id != carrier
+                )
+            )
+        ):
+            self.partner_shipping_id = False
+        if carrier and carrier.with_dropoff_site:
+            return {
```
Comment (rousseldenis) [inline delivery_dropoff_site/models/sale_order.py]: @jdoutreloux Returning domains is deprecated. Could you find another solution ?

Comment (jdoutreloux) [reply delivery_dropoff_site/models/sale_order.py]: @rousseldenis fixed it. Thanks for the catch :)

**Comment by DorianMAG** [comment]
Hi @jdoutreloux 
Can you rebase plz?

**Comment by ThomasBinsfeld** [comment]
> Hi @jdoutreloux Can you rebase plz?

Done

**Review by rousseldenis — APPROVED** [review]

**Comment by jdoutreloux** [comment]
Thanks for the rebase @ThomasBinsfeld

**Pull request merged**